manifests/m/Microsoft/Sysinternals/ProcessMonitor/3.92/Microsoft.Sysinternals.ProcessMonitor.installer.yaml

```diff
@@ -5,7 +5,7 @@ PackageIdentifier: Microsoft.Sysinternals.ProcessMonitor
 PackageVersion: 3.92
 InstallerType: portable
 Commands:
-- process-monitor
+- process-monitor /accepteula

```

Comment:
`Commands` isn't the right field for this, as it will cause issues on the CLI renaming the file